front/app/containers/InitiativesShowPage/index.tsx

```diff
@@ -35,6 +35,10 @@ const InitiativesShowPage = () => {
 
   const { data: initiative, status, error } = useInitiativeBySlug(slug);
 
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [initiative]);
```

Comment:
I wonder why we need this in the first place. Have you looked for code that causes the "jump"?

Reply:
It happens on multiple types of URLs, not just on InitiativesShowPage.

Reply:
Sure. If (on mobile at least) you go from home page to proposals overview (/initiatives) through clicking the proposals CTA on the home page, you have a similar bounce.